d2l-table-wrapper.js

```diff
@@ -49,10 +48,7 @@ Custom property | Description | Default
 `--d2l-table-border` | Border | `1px solid var(--d2l-table-border-color);` |
 `--d2l-table-header-background-color` | Header background color (th elements under `<thead>` or `<tr header>`) | `var(--d2l-color-regolith);` |
 `--d2l-table-body-background-color` | Body background color (non-header) | `#fff` |
-`--d2l-table-row-background-color-active` | Active row background color | `var(--d2l-color-celestine-plus-2)` |
 `--d2l-table-row-border-color-selected` | Selected row border color | `var(--d2l-color-celestine-plus-1)` |
-`--d2l-table-row-border-color-active-selected` | Active and Selected row border color | `var(--d2l-color-celestine-plus-1)` |
-`--d2l-table-row-background-color-active-selected` | Active and Selected row background color | `#EBF5FC` |
```

Comment:
I've removed the documentation for these variables, but there's still some code using them that I'll be cleaning up in a subsequent pull request when I also kill the concept of `selectable`.